utilities/tree.metta

```diff
@@ -30,6 +30,15 @@
             (mkTree (mkNode $expr) Nil))
             )
 
+(: cleanTree (-> (Tree $a) (Tree $a)))
+(= (cleanTree (mkTree (mkNode $r) (Cons $x $xs)))
+    (let*
+        (
+          ($preordered (preOrder (mkTree (mkNode $r) (Cons $x $xs))))
+          ($reduced (REDUCE $preordered))
```

Comment:
You should use the `reduce` function directly instead of a `REDUCE` function which is defined in the test case file. You should either move the `REDUCE` function here if it's existence has some relevance or replace it's use by the `reduce` function.

Reply:
I see, okay I get it know :)